chia/cmds/cmd_helpers.py

```diff
@@ -33,8 +33,7 @@ class NeedsWalletRPC:
         "-wp",
         "--wallet-rpc_port",
         help=(
-            "Set the port where the Wallet is hosting the RPC interface."
-            "See the rpc_port under wallet in config.yaml."
+            "Set the port where the Wallet is hosting the RPC interface.See the rpc_port under wallet in config.yaml."
```

Comment:
Fix the lack of spaces while we're in here?  or not.  feel free to resolve this conversation.  and maybe follow up separately.

Reply:
```suggestion
            "Set the port where the Wallet is hosting the RPC interface. See the rpc_port under wallet in config.yaml."
```